compiler/src/iree/compiler/Dialect/VM/Conversion/ArithToVM/Patterns.cpp

```diff
@@ -1,10 +1,10 @@
-// Copyright 2019 The IREE Authors
+// Copyright 2024 The IREE Authors
```

Comment:
Time flies :O

Reply:
right?! guess I'm somewhat proud that this code hasn't had to change much in that span, but also thought it'd be done by now :P